app/views/external_authors/_external_author_blurb.html.erb

```diff
@@ -11,7 +11,7 @@
       </li>
       -->
     <% end %>
-    <% if permit?("opendoors") %>
+    <% if permit?("opendoors", :get_user_method => 'current_user') %>
```

Comment:
Hound doesn't run on .html.erb files, so it didn't mention that you should probably change to the `get_user_method: 'current_user'` syntax here as well.